index.html

```diff
@@ -3,6 +3,12 @@
 title: Elixir & Phoenix Workshops
 ---
 <a name="top"></a>
+
+<div class="row header">
+    <span>Brought to you by </span>
+    <i class="simplabs"></i>
```

Comment:
This should link to https://simplabs.com with `target="_blank"`; also 2 spaces for indentation please ;)